src/styles/InstructorStyles/index.less

```diff
@@ -39,15 +39,13 @@
 }
 
 .dashboard-banner {
-  background-color: #006c72; //  orange colorFFAD00
```

Comment:
Commented-out styles if not needed should probably be removed before merging with main.